<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageDialog</class>
 <widget class="QDialog" name="ManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ManageDialog{
border-image:url(:/icon/icon/timg.jpg);
}
#ManageDialog *{
border-image:url();
}</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>520</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>取消/返回</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="manageTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>760</width>
     <height>490</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="goodsManagePage">
    <attribute name="title">
     <string>商品管理</string>
    </attribute>
    <widget class="QTableView" name="GoodsTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>740</width>
       <height>411</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="addGoodsButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>新添商品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delGoodsButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>430</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>430</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255,135,0);</string>
     </property>
     <property name="text">
      <string>修改商品信息，在表格中点击编辑即可</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="onSaleSetPage">
    <attribute name="title">
     <string>折扣与满减</string>
    </attribute>
    <widget class="QPushButton" name="discountSetButton">
     <property name="geometry">
      <rect>
       <x>109</x>
       <y>270</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>折扣状态修改</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>20</width>
       <height>451</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="onSaleCheckBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 135, 0);</string>
     </property>
     <property name="text">
      <string>满减活动开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="onSaleSetButton">
     <property name="geometry">
      <rect>
       <x>499</x>
       <y>270</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
background-color: rgb(255, 135, 0);
border-style: outset;
padding: 6px;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>满减状态修改</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>141</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 135, 0);</string>
          </property>
          <property name="text">
           <string>图书：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bookRateLineEdit">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 135, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 135, 0);</string>
          </property>
          <property name="text">
           <string>服装：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="clothRateLineEdit">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 135, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 135, 0);</string>
          </property>
          <property name="text">
           <string>食品：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="foodRateLineEdit">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 135, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 135, 0);</string>
          </property>
          <property name="text">
           <string>手机：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phoneRateLineEdit">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 135, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 135, 0);</string>
        </property>
        <property name="text">
         <string>全场满</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fullLineEdit">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 135, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 135, 0);</string>
        </property>
        <property name="text">
         <string>减</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="reduceLineEdit">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 135, 0);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>1</y>
       <width>761</width>
       <height>471</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/icon/timg.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_7</zorder>
    <zorder>discountSetButton</zorder>
    <zorder>line</zorder>
    <zorder>onSaleCheckBox</zorder>
    <zorder>onSaleSetButton</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget1</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
